lib/usbd/usbd_transfer.c

```diff
@@ -720,7 +720,7 @@ static void _control_data_callback(usbd_device *dev,
 		return;
 	}
 
-	usbd_control_transfer_callback callback = transfer->user_data;
+	usbd_control_transfer_callback callback = (usbd_control_transfer_callback)transfer->user_data;
```

Comment:
C standard says: void * will automatically cast to any other type.
and this style is more maintanable.
see: http://stackoverflow.com/a/605858/1500988
note to myself: place link to C standard text.